src/Drivers/StatamicTinkerwellDriver.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+use Tinkerwell\ContextMenu\SetCode;
+use Tinkerwell\ContextMenu\Submenu;
+
+class StatamicTinkerwellDriver extends TinkerwellDriver
+{
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/vendor/statamic/cms/src/Statamic.php') &&
```

Comment:
Probably just checking for vendor/statamic/cms is enough. Don’t need to check that specific php file, or please.